### ISSUE TITLE
Strange TLDs

### ISSUE THREAD

**Issue by mfpopa** [131/node-tld#6]
Hi there, I am using this library to identify TLDs in a web application, but I came across something I find strange.

Why does the `effective_tld_names.json` contain stirng like `compute-1.amazonaws.com` and `ap-northeast-1.elasticbeanstalk.com`? I would not consider these TLDs.

I expect the output for [http://ec2-23-20-71-128.compute-1.amazonaws.com/news/uk-news/2020/04/14/life-on-the-inside-10-ways-to-ease-an-anxious-mind-during-lockdown/](http://ec2-23-20-71-128.compute-1.amazonaws.com/news/uk-news/2020/04/14/life-on-the-inside-10-ways-to-ease-an-anxious-mind-during-lockdown/) to be:

```JS
{
    tld: 'com',
    domain: 'amazonaws.com',
    sub: 'ec2-23-20-71-128.compute-1',
};
```

Is there a way to exclude these from the list other than forking the library?

**Comment by 131** [comment]
I've asked clarification on the public suffix list official repo https://github.com/publicsuffix/list/pull/259, i hope they'll provide some answer

**Comment by 131** [comment]
Fixed in tld-extract@2.0.0
